modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -125,6 +128,25 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
       storages.block.store(fb.hashedBlock).handleErrorWith(e => logger.debug(e)("Block storing failed."))
     }
 
+  private val sendBlockToL0: Pipe[F, FinalBlock, Unit] =
+    _.evalMap { fb =>
+      for {
+        tips <- storages.block
+          .pullTips(appConfig.tips.minimumTipsCount)
```

Comment:
I think you could send just a single tip height to L0 instead of 2 block references.

Reply:
I would even create a function in `BlockStorage` for getting `getMinTipHeight: [Option[Height]]`

Reply:
I will talk about it with Tomek.